packages/fast-components/src/design-system-provider/design-system-provider.ts

```diff
@@ -0,0 +1,72 @@
+import { attr, FastElement, observable, Observable } from "@microsoft/fast-element";
+
+/**
+ * The following interface is an implementation of
+ * design-system.schema.json in TypeScript with
+ * as much accuracy as TypeScript supports.
+ */
+export interface DesignSystemData {
+    accentBaseColor: string;
+    backgroundColor: string;
+    cornerRadius: number;
+    disabledOpacity: number;
+    density: number;
+    designUnit: number;
+    elevatedCornerRadius: number;
+    focusOutlineWidth: number;
+    outlineWidth: number;
+}
+
+function camelToKebab(str: string): string {

```

Comment:
we have a "spinalCase" function in fast-web-utilities, should we just improve that?

Reply:
I'd prefer to wait as this is a temporary implementation and I don't want to spend the time right now writing an extensive function w/ full tests.